services/src/main/scala/cromwell/services/metadata/impl/MetadataServiceActor.scala

```diff
@@ -58,6 +64,14 @@ case class MetadataServiceActor(serviceConfig: Config, globalConfig: Config, ser
   private val metadataReadRowNumberSafetyThreshold: Int =
     serviceConfig.getOrElse[Int]("metadata-read-row-number-safety-threshold", 1000000)
 
+  private val metadataTableMetricsInitialDelay = serviceConfig.getOrElse[FiniteDuration]("table-size-metrics-initial-delay", 5.minutes)
```

Comment:
In the archiver we decided not to bother with an initial delay and just jump straight into archiving. Any reason here not to jump straight into monitoring?

Reply:
My only reason was to give enough time for `ServiceRegistoryActor` to start up `MetadataServiceActor`.  If initial delay feels unnecessary then I can remove it.

Reply:
Doesn't hurt, so 👍  as far as I'm concerned